scripts/us_eia/opendata/process/elec.py

```diff
@@ -216,22 +216,22 @@ def extract_place_statvar(series_id, counters):
 # Unit with this value is handled specially depending on the fuel type.
 _PLACEHOLDER_FUEL_UNIT = '_XYZ_'
 
-# The value is a pair of (unit, scalingFactor) for each measure.
+# The value is a pair of (unit, scalingFactor, multiplier) for each measure.
 _UNIT_MAP = {
-    'ASH_CONTENT': ('', '100'),
-    'CONS_EG': (_PLACEHOLDER_FUEL_UNIT, '1000'),
-    'CONS_EG_BTU': ('MMBtu', '1000000'),
-    'COST': (_PLACEHOLDER_FUEL_UNIT, ''),
-    'COST_BTU': ('MMBtu', ''),
-    'CUSTOMERS': ('', ''),
-    'GEN': ('MegawattHour', '1000'),
-    'PRICE': ('USCentPerKilowattHour', ''),
-    'RECEIPTS': (_PLACEHOLDER_FUEL_UNIT, '1000'),
-    'RECEIPTS_BTU': ('Btu', '1000000000'),
-    'REV': ('USDollar', '1000000'),
-    'SALES': ('KilowattHour', '1000000'),
-    'STOCKS': (_PLACEHOLDER_FUEL_UNIT, '1000'),
-    'SULFUR_CONTENT': ('', '100'),
+    'ASH_CONTENT': ('', '100', 1),
```

Comment:
should we use https://datacommons.org/browser/Percent as a unit for this?

Reply:
This I think used to be a legacy representation. So I'm slightly hesitant to use so that we're not adding one more slightly different way of representing percent.

For context, we have the following representations for percent:
1. unit = Percent
2. measuredProp = percent
3. measurementDenominator = X, scalingFactor = 100

We sort of deprecated (1) and (2) and moved to approach (3) in the more recent imports.

In (3), if instead of (or in addition to?) scalingFactor, we do unit = Percent, that will be another new way.